### ISSUE TITLE
Questions regarding to the PSNR/SSIM results

### ISSUE THREAD

**Issue by crisb-DUT** [cchen156/Learning-to-See-in-the-Dark#23]
Hi @cchen156 , 

This work is very exciting. But I have a question about the quantitative results reported in the paper Table3.

I downloaded the source code and trained model, and tested Sony dataset. I generate GT by Rawpy and apply Gamma transfer after it, since you apply Gamma transfer on the output of the network in your test code. I found that the results looks fine, but the average PSNR/SSIM in the testset is 25.44/0.713, which is quite lower than 28.88/0.787 reported in Table3 in your paper. 

If I have made some mistake? How can I get the reported results.

Thanks,
Wenbo

**Comment by cchen156** [comment]
Hi, rawpy already applied gamma correction by default. I think you applied gamma correction twice in this case.

**Comment by eedalong** [comment]
wait, where is the gamma correction ?  i didnt do any gamma correction and i fasten the training procedure by 4 times .I trained the model for 1 hour and test on full test set ,and the psnr is 30.276dB

**Comment by eedalong** [comment]
and the results look pretty good

**Comment by cchen156** [comment]
The rawpy. postprocess contains gamma correction. 

How did you train it with 1 hour and get 30.2dB psnr. This is interesting!

**Comment by eedalong** [comment]
well ,
1 . i train on a mini dataset which only contains 110 sampels.(just training inputs 0-110 in ur train.txt),and trained for 400 epochs 
2. in ur codes , u generate  one sample from one input  ,u save all the inputs in RAM, i generate 8 from one input and batchsize  is 2 so there are few io requires, i dont save them in RAM  
3. i use 2 TITANX  GPUS
4. i reimplemented ur codes in pytorch (i have no idea whether this has any influence on the final results );

**Comment by eedalong** [comment]
actually , i have reimplemented a paper "DeepISP: Learning End-to-End Image Processing Pipeline",they released a dataset as well ,and i think their dataset has no ensential difference with urs ,their training dataset contrains ~100 samples and they achieve a pretty result (at least they claimed that their results are good ).their model has  more parameters then SIDNet ,so i thought 100 samples might be enough for training this model

**Comment by eedalong** [comment]
i reimplemented the paper wth pytorch,and i replaced the last layer with a grouped decov layer @crisb-DUT

**Comment by xiaojidan** [comment]
@eedalong  can you share your code?

**Comment by KeqiWangSXuniversity** [comment]
hello， @cchen156 
I also encountered such a problem. The average PSNR/SSIM in the Sony testset is 28.64/0.7674., which is lower than 28.88/0.787 reported in Table3 in your paper.I downloaded the model you provided, but the results were very poor. I felt that the model could not be loaded correctly, but I checked my path and there was no problem.As with the code provided by your github, no changes have been made.English is not very good, please forgive me.

**Comment by KeqiWangSXuniversity** [comment]
Hello, I downloaded the ckpt you only have two files, model.ckpt.data-00000-of-00001, model.ckpt.meta, is my download error, or what caused it? Should there be four files, and should also include checkpoint, model.ckpt.index. Due to the lack of these two files, this model is difficult to load into, is it really a problem with my operation?

**Comment by KeqiWangSXuniversity** [comment]
> Hi can you try the Baidu link: https://pan.baidu.com/s/1WfijVSQfBimQLCU_QL_Iaw

Ok, thank you

**Comment by lyn-max** [comment]
@KeqiWangSXuniversity Hi, have you solved the pretrained-model problem?
